include/maths/CBoostedTree.h

```diff
@@ -291,6 +318,8 @@ class MATHS_EXPORT CBoostedTree final : public CDataFrameRegressionModel {
     //! Populate the object from serialized data.
     bool acceptRestoreTraverser(core::CStateRestoreTraverser& traverser);
 
+    void accept(Visitor& visitor);
```

Comment:
Couldn't we keep this const
```suggestion
    void accept(CVisitor& visitor) const;
```

Reply:
Done